.github/workflows/release.yaml

```diff
@@ -66,10 +66,10 @@ jobs:
           persist-credentials: true
 
       - name: Bootstrap environment
-        uses: ./.github/actions/bootstrap
+        uses: anchore/go-make/.github/actions/bootstrap@package-split
```

Comment:
Need to update all these references before merging -- what should it be `@main`? Should we release with `latest` tags, and just use `@latest` so we use the last release?

Reply:
@latest makes a lot of sense. If we do @vX we'll need to bump this everywhere, if we do @main, we won't be able to go back to original functionality without reverting/committing. With @latest we can roll forward and backwards by tagging and deleting the tag.